frontend/src/scenes/notebooks/Nodes/NodeWrapper.scss

```diff
@@ -13,10 +13,10 @@
 
     transition: outline 150ms ease-out, outline-offset 150ms ease-out;
 
-    &--selected {
-        outline: 2px solid var(--accent-3000-light);
-        outline-offset: 0.25rem;
-    }
+    // &--selected {
+    //     outline: 2px solid var(--accent-3000-light);
+    //     outline-offset: 0.25rem;
+    // }
```

Comment:
Commented out code isn't ideal

Reply:
This was a bad commit from swapping branches too quick 🙈